src/Adapters/Storage/Mongo/MongoCollection.js

```diff
@@ -160,18 +160,10 @@ export default class MongoCollection {
   }
 
   _ensureSparseUniqueIndexInBackground(indexRequest) {
-    return new Promise((resolve, reject) => {
-      this._mongoCollection.createIndex(
-        indexRequest,
-        { unique: true, background: true, sparse: true },
-        error => {
-          if (error) {
-            reject(error);
-          } else {
-            resolve();
-          }
-        }
-      );
+    return this._mongoCollection.createIndex(indexRequest, {
```

Comment:
Does this return a promise? I assume the reason why this was wrapped in a promise previously was that it wasn't returning a promise?

Reply:
Yes in the mongo v4 version, they use callback so we needed Promise, but in the mongo v5 the callback is removed so we don't need anymore new Promise :)